examples/bri-1/lib/proubc-abap-connector/PROUBC-intro.md

```diff
@@ -0,0 +1,21 @@
+# proUBC
+*An open source, SAP ABAP connector for PRVD Baseline aka BRI-1*
+
+proUBC is an open source connector to make Baseline Protocol integration possible in SAP ABAP systems - such as S/4 HANA or SAP BTP ABAP Environment
+
+A link to the proUBC repository is provided [here](https://github.com/provideplatform/proUBC.git)
+
+This repository is maintained via abapGit. Instructions on how to clone, fork and maintain abapGit repositories such as this one can be found [here](https://docs.abapgit.org/)
+
+Additional post-install, configuration, testing, and other instructions for using proUBC are provided in proUBC repository's README.
+
+The scope of proUBC's present implementation of Baseline includes:
+- Onboarding Baseline users via the proUBC REST component (invoked by Shuttle as a UX)
+- Authenticating and linking DIDs/VCs used by BRI-1 to SAP Credentials via PRVD Ident component
+- Initiating a Baseline Protocol message via the PRVD Baseline component
+
+A sample integration was created using the batch program `ZPROUBC_SHUTTLEIDOC_EXMPL` to create protocol messages from SAP Purchase Orders.
+
+The batch program takes a PRVD Ident Subject Account ID + Tenant ID as authentication input and prepares a Baseline Protocol message based on an a selected ORDERS05 or ORDERS02 iDoc from SAP. iDocs - for those less familiar - are a standard EDI message format used by SAP systems globally for purchase orders / sales orders.
```

Comment:
is there a URL for ORDERS05 or ORDERS02 that can be added here for a quick reference?
Also, what does Tenant ID refer to? Can we have a little context?

Reply:
Hi the links below provide SAP's documentation on the ORDERS02 standard
https://help.sap.com/docs/SAP_ERP/61155ee7ed304ae8b66b7526555a6a07/7264b6531de6b64ce10000000a174cb4.html?version=6.18.latest&q=orders02
and for ORDERS05
https://help.sap.com/docs/SAP_ERP/349992fb60854a62a264d716ad5c8f54/cc9a62543da42357e10000000a44176d.html?q=orders05

I can include these links for further reference

Concerning Tenant ID - it's one of the identifiers along with PRVD Subject account ID that uniquely identifies the user to Baseline. Let me know if you need more info on how the SAP user id this are linked - Thanks for the feedback!

Reply:
Support for multi-tenancy is the context.